climada/engine/impact.py

```diff
@@ -150,6 +153,7 @@ def __init__(self,
         self.crs = crs
         self.eai_exp = np.array([], float) if eai_exp is None else eai_exp
         self.at_event = np.array([], float) if at_event is None else at_event
+        self.at_reg_id_event = np.array([], float) if at_reg_id_event is None else at_reg_id_event
```

Comment:
I don't understand why this should become an attribute of `Impact`. If it can be calculated from `Impact`, is there any case where you could *create* an `Impact` with it? I would make the function *only* return the value, but not store it in the `Impact` object. The user than has to take care of storing it.

Reply:
I am quite fine with that and will make the change. But you can't _create_ an `Impact` from `eai_exp`, `at_event` and many others either, can you?

Reply:
removed from attributes